STEER/STEERBase/AliTPCPIDResponse.h

```diff
@@ -173,27 +173,27 @@ class AliTPCPIDResponse: public TNamed {
                               ETPCdEdxSource dedxSource = kdEdxDefault,
                               Bool_t correctEta = kFALSE,
                               Bool_t correctMultiplicity = kFALSE,
-                              Bool_t usePileupCorrection = kFALSE) const;
+                              Bool_t usePileupCorrection = kFALSE, Bool_t useQPtTglCorrection = kFALSE) const;
   Double_t GetExpectedSigma( const AliVTrack* track, 
                              AliPID::EParticleType species,
                              ETPCdEdxSource dedxSource = kdEdxDefault,
                              Bool_t correctEta = kFALSE,
                              Bool_t correctMultiplicity = kFALSE,
-                             Bool_t usePileupCorrection = kFALSE) const;
+                             Bool_t usePileupCorrection = kFALSE, Bool_t useQPtTglCorrection = kFALSE) const;
   Float_t GetNumberOfSigmas( const AliVTrack* track,
                              AliPID::EParticleType species,
                              ETPCdEdxSource dedxSource = kdEdxDefault,
                              Bool_t correctEta = kFALSE,
                              Bool_t correctMultiplicity = kFALSE,
-                             Bool_t usePileupCorrection = kFALSE) const;
+                             Bool_t usePileupCorrection = kFALSE, Bool_t useQPtTglCorrection = kFALSE) const;
   
   Float_t GetSignalDelta( const AliVTrack* track,
                           AliPID::EParticleType species,
                           ETPCdEdxSource dedxSource = kdEdxDefault,
                           Bool_t correctEta = kFALSE,
                           Bool_t correctMultiplicity = kFALSE,
                           Bool_t usePileupCorrection = kFALSE,
-                          Bool_t ratio = kFALSE) const;
+                          Bool_t ratio = kFALSE, Bool_t useQPtTglCorrection = kFALSE) const;
```

Comment:
Above you have a different order, first `Bool_t useQPtTglCorrection`, then `Bool_t ratio`.